implementations/rust/ockam/ockam/src/remote.rs

```diff
@@ -283,32 +283,28 @@ mod test {
     #[allow(non_snake_case)]
     #[ockam_macros::test]
     async fn forwarding__ephemeral_address__should_respond(ctx: &mut Context) -> Result<()> {
-        let cloud_address;
-        if let Some(c) = get_cloud_address() {
-            cloud_address = c;
-        } else {
-            ctx.stop().await?;
-            return Ok(());
-        }
+        let cloud_address = match get_cloud_address() {
+            Some(c) => c,
+            None => {
+                ctx.stop().await?;
+                return Ok(());
+            }
+        };
 
         ctx.start_worker("echoer", Echoer).await?;
 
-        TcpTransport::create(&ctx).await?;
+        TcpTransport::create(ctx).await?;
 
         let node_in_hub = (TCP, cloud_address);
         let remote_info = RemoteForwarder::create(ctx, node_in_hub.clone()).await?;
 
-        let mut child_ctx = ctx.new_context(Address::random_local()).await?;
-
-        child_ctx
-            .send(
+        let resp: String = ctx
+            .send_and_receive(
```

Comment:
Nothing wrong with this, but can you please use a turbofish for consistency with the rest of the codebase?

i.e.

Instead of:

`let resp: String = ctx.send_and_receive(...)`

Use:

`let resp = ctx.send_and_receive::<String>(...)`

Reply:
Sure. I've updated the PR. Thanks!